tasks/publish-pyxis-repository/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# Add mocks to the beginning of task step script
+yq -i '.spec.steps[0].script = load_str("'$SCRIPT_DIR'/mocks.sh") + .spec.steps[0].script' $SCRIPT_DIR/../publish-pyxis-repository.yaml
+
+# Create a dummy pyxis secret (and delete it first if it exists)
+kubectl delete secret test-create-pyxis-image-cert --ignore-not-found
```

Comment:
looks like we use the task name in the secret name. This is using a different task name. Likely just forgot to change it

Reply:
good catch, fixed